### DISCUSSION TITLE
Question on Purpose definition, W3C Data Privacy Vocabulary​ (DPV)

### DISCUSSION THREAD

**Discussion by ToshiWakayama-KDDI** [camaraproject/IdentityAndConsentManagement#222] [Q&A] [answered]
CAMARA APIs Access and User Consent Management has 'Purpose definition' which states the following:
The purpose definition (naming + description) and format for CAMARA follows the W3C Data Privacy Vocabulary​ (DPV). The way in which purposes are organised within the DPV is specified in Purpose taxonomy in DPV.

(Links)
* CAMARA APIs Access and User Consent Management https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-API-access-and-user-consent.md#camara-apis-access-and-user-consent-management

* Purpose definition: https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-API-access-and-user-consent.md#purpose-definition

* Data Privacy Vocabulary​ (DPV): https://w3c.github.io/dpv/2.0/dpv/  

**Question:** 
the W3C Data Privacy Vocabulary (DPV) above is just a Community Report, not specifications.  It is understood that it is possible for CAMARA to refer to a W3C report, however, it is better to refer to specifications.  Does anyone know if W3C is going to make DPV specifications in near future?

Thanks.
Toshi
KDDI

**Comment by ToshiWakayama-KDDI** [comment]
Hi Community, 

There has not been any answer yet, so, please let me reiterate the question with my problem. 

Company-internally from implementation side, I have received a comment that **DPV's status is currently W3C Report, not W3C Standard**, and my company's implementation side cannot be convinced to implement DPV.  **Does anyone know if there is any plan to standardise DPV in W3C?  More specifically, has your company implemented DPV or decided to implement DPV before it is standardised?**


I understand that DPV is specified in the ICM document here:
https://w3c.github.io/dpv/2.0/dpv/

This link leads me to the W3C DPV v2.0 page, where I can see W3C Community Group Final Report.

According to the W3C definition of 'types' https://www.w3.org/standards/types/, 

- Section 2.1 defines 'W3C Community Group Report' https://www.w3.org/standards/types/#x2-1-w3c-community-group-report-or-w3c-business-group-report .
- Section 5 defines 'W3C Standard' https://www.w3.org/standards/types/#x5-standard

Regarding W3C Community Group Report, Section 2.1 has the following text:

> These reports MUST NOT be cited as W3C standards. They may or may not be brought to the W3C standardization track.
> 
> Software MAY implement these reports at their own risk. Implementation is neither discouraged nor encouraged but can contribute to proposals for further action on the document.
> 

Currently DPV is W3C Community Group Report, and it is said 'Software MAY implement these reports at their own risk'.  So, my company's implementation side cannot be convinced to implement DPV.  **Does anyone know if there is any plan to standardise DPV in W3C?  More specifically, has your company implemented DPV or decided to implement DPV before it is standardised?**

Or, please let me know if my understanding above has anything wrong.

Many thanks,
Toshi

**Comment by diegogonmar** [comment, marked as answer]
Hi @ToshiWakayama-KDDI ,
I don't know the plan in W3C to move from Report to Standard, sorry.
I can confirm that we are using it in Telefónica; as CAMARA has taken it as a reference, we must support it to be CAMARA compliant. We're not very worried about potential changes as we think most of the values won't change. If few new values apply, we can just start using those in the APIs without relevant impacts.
Kind regards,
    Diego

**Comment by eric-murray** [comment]
> So, my company's implementation side cannot be convinced to implement DPV.

The majority of [IETF RFCs](https://www.rfc-editor.org/rfc-index.html) remain formally either DRAFT or PROPOSED standards. So hopefully your company is not implementing any of these either :-)

Of course, it is known that documents and even standards can evolve. What is important is that CAMARA refers to specific versions of documents that remain stable, in this case v2 of the Data Privacy Vocabulary. W3C may produce later versions of this report, and CAMARA must then decide whether to use these later versions, or remain using v2 as the reference for the CAMARA standard. But that is under the control of CAMARA - v2 will remain available as a reference even if superseded.
